src/views/event-detail/components/SpeakerSection.tsx

```diff
@@ -0,0 +1,21 @@
+import { Stack } from '@mui/material'
+import { Children, FC } from 'react'
+import { SpeakerCard } from '~/components/SpeakerCard'
+import { EventDetailWrapper } from './EventDetailWrapper'
+import { Program } from '~/features/program/types'
+
+type SpeakerSectionProps = Pick<Program, 'speakers'>
+
+export const SpeakerSection: FC<SpeakerSectionProps> = ({ speakers }) => {
+  return (
+    <EventDetailWrapper title="Tentang Instruktur" isBorder={false}>
+      <Stack spacing="1rem" mt="1rem">
+        {Children.toArray(
+          speakers?.map((speaker) => (
+            <SpeakerCard key={speaker.name} {...speaker} showDescription />
+          ))
+        )}
```

Comment:
```suggestion
        {Children.toArray(
          speakers?.map((speaker) => (
            <SpeakerCard {...speaker} showDescription />
          ))
        )}
```
kalau udah pake Children.toArray gak perlu pasang key lagi bukan?